app/feemarket.go

```diff
@@ -0,0 +1,50 @@
+package app
+
+import (
+	"fmt"
+
+	storetypes "cosmossdk.io/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	feemarket "github.com/skip-mev/feemarket/x/feemarket"
+	feemarketkeeper "github.com/skip-mev/feemarket/x/feemarket/keeper"
+	feemarkettypes "github.com/skip-mev/feemarket/x/feemarket/types"
+)
+
+// registerFeeMarketModule registers the feemarket module.
+func (app *AlloraApp) registerFeeMarketModule() {
+	if err := app.RegisterStores(
```

Comment:
This logic could have been added to `legacy.go` with ibc wiring, no problem having it separated though :)

Reply:
oh yeah, I agree!

Reply:
We don't need to add this interface registration that you mentioned; we already register interfaces during the `RegisterModules` function.